support-metrics-common/src/main/java/io/confluent/support/metrics/common/kafka/KafkaUtilities.java

```diff
@@ -265,8 +264,7 @@ public VerifyTopicState verifySupportTopic(ZkUtils zkUtils,
   }
 
   public boolean isReadyForMetricsCollection(KafkaServer server) {
-    return server.brokerState().currentState() == RunningAsBroker.state() ||
-        server.brokerState().currentState() == RunningAsController.state();
+    return server.brokerState().currentState() == RunningAsBroker.state();
```

Comment:
Does this mean you are able to create topics and write to it ? Do you not need controller leader election to be over before the cluster is ready to accept client requests?